README.md

```diff
@@ -1,56 +1,45 @@
+After submitting your code, you will be invited to our offices to discuss your choices, what you prioritised and the next steps you’d take. Your code will be **the starting point for a conversation**, and it will be helpful if you could briefly highlight reasons for  your architectural choices in a README in your repo.
 
-### http-server
+## The rules
 
-To run the challenge on your localhost you'll need to install the dependency `http-server` globally:
+When completing this challenge, we encourage you to use vanilla JavaScript, HTML and CSS. You are free to use appropriate modern JavaScript libraries, build tooling and boilerplates if you prefer. 
 
-`npm install -g http-server`
+However, please don't use:
 
-Once you've installed this you can run the command `npm run start` and open `http://localhost:8000` in your browser.
+- the Angular framework
+- any CSS framework such as Bootstrap or Material Design
 
-## Instructions
-
-Thanks for your interest in applying for the role. We’d like you to complete a coding task to see your coding approach, learn how you structure a solution and see what kind of things you can do when allowed to be creative.
-
-We’ve set the deadline at two hours: we’re interested in how you would build a production-ready feature. Two hours may not be long enough to finish. Your code will be the starting point for a conversation about your choices, what you prioritised and any next-steps you’d take. Feel free to use comments or add a readme.
+These frameworks make it difficult for us to assess your code fairly.
 
 ## The exercise
 
-Inside this repo is an incomplete **tabbed component**. We'd like you to pick up this piece of work and update it so it meets the requirements listed below.
-
-## Requirements
-
-The requirements have been split into manadatory and optional groups. Please take time to read the requirements carefully. We'd like you to have a go at achieving the mandatory requirements and if you have the time you can then choose 2 optional requirements to focus on.
-
-### Mandatory requirements
+Inside this repo is an incomplete **tabbed component**. We'd like you to update it to meet the requirements listed below. 
 
 You should implement a **tabbed component** that looks similar to the Guardian’s most recent content widget. The component will be a numbered listing of the most recent Guardian content chosen from three sections of the site: UK news, Football and Travel. Each of these sections should be represented by a tab.
 
-We’d like you to fetch this data asynchronously from our [Content API](https://open-platform.theguardian.com/explore/). Please do this on the client-side, your solution should run in the browser alone with no server-side requirements.
+You should fetch this data asynchronously from our [Content API](https://open-platform.theguardian.com/explore/). **Please do this on the client-side.** Your solution should run in the browser alone with no server-side requirements.
```

Comment:
Perhaps this wording need clarification that **we are not ruling out build tooling**. What do you think?

Reply:
👍